src/components/gallery/GalleryContainer.styles.ts

```diff
@@ -1,16 +1,16 @@
 import styled from 'styled-components'
-import { layout, LayoutProps, space, SpaceProps } from 'styled-system'
-import { sm } from '../../theme/space'
+import { media } from '../../theme/media'
 
-export const Grid = styled.div<LayoutProps & SpaceProps>`
-  display: flex;
-  flex-wrap: wrap;
-
-  > * {
-    padding: ${sm}px;
-
-    ${layout}
-  }
-
-  ${space}
+export const Grid = styled.div`
+  display: grid;
+  grid-gap: 1.6rem;
+  ${media.mobile`
+    grid-template-columns: repeat(1, 1fr);
+  `}
+  ${media.tablet`
+    grid-template-columns: repeat(2, 1fr);
+  `}
+  ${media.desktop`
+    grid-template-columns: repeat(3, 1fr);
+  `}
```

Comment:
This is not the same as previous implementation and requested design spec - please can this be reverted.

Reply:
I've reverted.

Reply:
I left the same category filter as the one on https://prideinlondon.org/fifty-two but I can revert if needed.